db/migrate/20160111013907_create_option_sets.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOptionSets < ActiveRecord::Migration
+  def change
+    create_table :option_sets do |t|
+      t.references :division, index: true, null: false, foreign_key: true
+      t.string :model_type
```

Comment:
is there built in polymorphic handling against just a class, (there is not associated mode instance in this case, which is the typical pattern)